### DISCUSSION TITLE
Guide button support?

### DISCUSSION THREAD

**Discussion by ReindeerFloatilla** [darthcloud/BlueRetro#768] [Ideas]
While I like using this on my PS2 I also would like to use it with a converter for my 360 and PS3. Since it's one of the only dongles that supports the Google Stadia controller. However with doing that the guide button doesn't get passed through.

Would it be possible to just pass through the guide button through a firmware update? I have used dongles that do pass through the guide button for other controllers. (Like the Retro Fighters Defender).

**Comment by darthcloud** [comment]
See mapping reference
https://docs.google.com/spreadsheets/d/e/2PACX-1vT9rPK2__komCjELFpf0UYz0cMWwvhAXgAU7C9nnwtgEaivjsh0q0xeCEiZAMA-paMrneePV7IqdX48/pubhtml

All statia button available but you will have to manually map the one unused on PS2 by default to as button that exist on PS2.
